packages/logs/src/domain/logger.ts

```diff
@@ -64,7 +65,7 @@ export class Logger {
 
     if (error !== undefined && error !== null) {
       const rawError = computeRawError({
-        stackTrace: error instanceof Error ? computeStackTrace(error) : undefined,
+        stackTrace: isError(error) ? computeStackTrace(error) : undefined,
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** Could be nice to have a test about it.‏